src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <header class="header">
+    <a href="main page" class="logo">
+      <img src="images/logo.png" alt="moyo logo">
+    </a>
+    <nav class="nav">
+      <ul class="menu">
+        <li class="nav_item"><a href="Apple" class="nav_link is-active">Apple</a></li>
```

Comment:
```suggestion
        <li class="nav_item">
          <a href="#Apple" class="nav_link is-active">
            Apple
          </a>
        </li>
```

Reply:
Fix all links, and make them anchor. Fix formatting. Check all other similar cases.